fixtures/jar-app/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/tmp-test-project-1380/openjdk8:testing
```

Comment:
Actually Dockerfile is not part of maven project resources. So, we can't easily use the resource filtering. I'll use `sed` instead.